templates/create-frontend-plugin/docs/features/api.md

```diff
@@ -0,0 +1,97 @@
+## Guide to consuming APIs in your UI components
+
+The ExampleFetchComponent demonstrates the common task of making an asynchronous request to a public API and displaying the response data in a table using Material UI components.
+
+You can modify these components, rename them, or replace them entirely.
+
+To consume APIs from your backend plugin, follow these steps:
+
+- Create an `api` folder under your `src` directory and add a client file for making API calls. The file name should follow the format `<PluginNameInTitleCase>BackendClient.ts`.
+
+    The file structure should be as follows:
+
+    ```ts title="<PluginNameInTitleCase>BackendClient.ts"
```

Comment:
Does &lg; &gt; works here? Because the < and > isn't rendered:

![image](https://github.com/user-attachments/assets/1f00481c-9e9b-4b0c-a80e-be110de7a5c0)